conda/cli/conda_argparse.py

```diff
@@ -199,6 +205,21 @@ def _check_value(self, action, value):
         else:
             super()._check_value(action, value)
 
+    def parse_args(self, args=None, namespace=None):
+        """
+        We override this method to check if we are running from a known plugin subcommand.
+        If we are, we do not want to handle argument parsing as this is delegated to the plugin
+        subcommand. We instead return a ``Namespace`` object with ``plugin_subcommand`` defined,
+        which is a ``conda.plugins.CondaSubcommand`` object.
+        """
+        if len(sys.argv) > 1:
+            name = sys.argv[1]
+            subcommand = next((sub for sub in self._subcommands if sub.name == name), None)
+            if subcommand:
+                return Namespace(plugin_subcommand=subcommand)
```

Comment:
A little easier to understand:

```suggestion
            for subcommand in self._subcommands:
                if subcommand.name == name:
                    return Namespace(plugin_subcommand=subcommand)
```

Do we need to pass `args` and `namespace` here, too?

Reply:
No we do not. The `parse_args` method only returns a single `Namespace` object.